Automattic/WP-Job-Manager
# Add/email notification settings

### Changes proposed in this Pull Request

* Currently in WPJM, you can have emails be sent to the main Admin account.  This is not always ideal as the Admin account might be someone who doesn't deal with jobs.  
* This PR adds a new setting for each "Admin" Email notification so you can customize the email address it sends to.
* If no email is set, it will default to the Admin email.

### Testing instructions

* Check out this PR
* Go to Job Listings > Settings > Email Notifications
* Enter email addresses for the 3 new settings (New Job, Updated Job, Expired Job)
* Use a plugin like Email Log to capture what emails are sent
* Trigger each email
* Ensure you see the emails sent to the new email addresses

### Screenshot / Video

<img width="985" alt="Screenshot 2023-03-20 at 1 34 28 PM" src="https://user-images.githubusercontent.com/3220162/226459994-f496492d-f322-4429-b774-9567a67c6661.png">
